GrandCentralBoard/App/ViewControllers/LoadingViewController.swift

```diff
@@ -5,20 +5,33 @@
 
 import UIKit
 import GrandCentralBoardCore
+import Alamofire
 
-private let dataDownloader = DataDownloader()
-private let availableBuilders: [WidgetBuilding] = [WatchWidgetBuilder(dataDownloader: dataDownloader), BonusWidgetBuilder(dataDownloader: dataDownloader)]
-private let useLocal = debugBuild
+
+private let debugBuild: Bool = _isDebugAssertConfiguration()
+
+// Set to `true` if you want to load bundled `configuration.json`.
+private let shouldUseLocalConfigurationOnDebugBuild = false
```

Comment:
Could we have maybe two schemes in the project, one which will fetch the configuration and one that will launch with provided config file? Setting it in code hides it, imo. We could then set parameters in the schemes.